src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -470,61 +457,67 @@ function EmojiPickerMenu(props) {
     const height = !listStyle.maxHeight || listStyle.height < listStyle.maxHeight ? listStyle.height : listStyle.maxHeight;
     const overflowLimit = Math.floor(height / CONST.EMOJI_PICKER_ITEM_HEIGHT) * 8;
     return (
-        <View
-            style={[
-                styles.emojiPickerContainer,
-                StyleUtils.getEmojiPickerStyle(isSmallScreenWidth),
-                // Disable pointer events so that onHover doesn't get triggered when the items move while we're scrolling
-                arePointerEventsDisabled ? styles.pointerEventsNone : styles.pointerEventsAuto,
-            ]}
-        >
-            <View style={[styles.ph4, styles.pb3, styles.pt2]}>
-                <TextInput
-                    label={translate('common.search')}
-                    accessibilityLabel={translate('common.search')}
-                    role={CONST.ACCESSIBILITY_ROLE.TEXT}
-                    onChangeText={filterEmojis}
-                    defaultValue=""
-                    ref={searchInputRef}
-                    autoFocus={shouldFocusInputOnScreenFocus}
-                    onSelectionChange={onSelectionChange}
-                    onFocus={() => {
-                        setHighlightedIndex(-1);
-                        setIsFocused(true);
-                        setIsUsingKeyboardMovement(false);
-                    }}
-                    onBlur={() => setIsFocused(false)}
-                    autoCorrect={false}
-                    blurOnSubmit={filteredEmojis.length > 0}
-                />
-            </View>
-            {!isFiltered && (
-                <CategoryShortcutBar
-                    headerEmojis={headerEmojis}
-                    onPress={scrollToHeader}
-                />
-            )}
-            <FlatList
-                ref={emojiListRef}
-                data={filteredEmojis}
-                renderItem={renderItem}
-                keyExtractor={keyExtractor}
-                numColumns={CONST.EMOJI_NUM_PER_ROW}
+        <View>
```

Comment:
What's happening here? Why we nest a `View` inside a `View` with no style or attributes?

Reply:
good catch, thank you!